src/plugins/ep_lightship/package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "ep_lightship",
+  "version": "0.1.0",
+  "description": "Etherpad light plugin that integrates lightship health checks",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "OpenFun",
+  "dependencies": {"lightship": "6.1.0"},
```

Comment:
```suggestion
  "dependencies": {
    "lightship": "6.1.0",
  },
```

Reply:
Prettier seems to have mixed feelings about this...